modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -38,6 +38,10 @@ If not set, the default value is based on `infrastructure.config.openshift.io/cl
 * Google Cloud Platform (GCP): `LoadBalancerService` (with External scope)
 * Bare metal: `NodePortService`
 * Other: `HostNetwork`
+
+On GCP, AWS, and Azure you can configure the following `endpointPublishingStrategy` fields:
```

Comment:
This isn't rendering right in the preview build. You need a line break after this line for the bullets to render properly.

Reply:
addressed